.github/workflows/ci.yaml

```diff
@@ -208,6 +208,33 @@ jobs:
             --batch-mode \
             --no-transfer-progress
 
+  showcase-native:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+      - uses: graalvm/setup-graalvm@v1
+        with:
+          version: '22.3.2'
+          java-version: '17'
+          components: 'native-image'
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - run: mvn -version
+      - run: native-image --version
+      - name: Install sdk-platform-java
+        run: mvn install -B -ntp -DskipTests -Dclirr.skip -Dcheckstyle.skip
+      - name: Install showcase server
+        run: |
+          sudo mkdir -p /usr/src/showcase
+          sudo chown -R ${USER} /usr/src/
+          curl --location https://github.com/googleapis/gapic-showcase/releases/download/v${SHOWCASE_VERSION}/gapic-showcase-${SHOWCASE_VERSION}-linux-amd64.tar.gz --output /usr/src/showcase/showcase-${SHOWCASE_VERSION}-linux-amd64.tar.gz
+          cd /usr/src/showcase/
+          tar -xf showcase-*
+          ./gapic-showcase run &
+          cd -
+      - name: Build native image
+        working-directory: showcase
+        run: mvn test -Pnative,-showcase -ntp -B
+
```

Comment:
@burkedavison I think we already run showcase tests with native image compilation in our Kokoro presubmits so I wonder if this is still needed? : https://github.com/googleapis/sdk-platform-java/blob/a377aab5149aedb55426a77335d38923a256ab28/.kokoro/presubmit/downstream-build.sh#L78

Reply:
Hm but it also looks like the `Kokoro - Test: Downstream Java 17 GraalVM Native Image` job hasn't been running in the most recent PRs.

Reply:
@mpeddada1 : I made the suggested fix, but the Kokoro builds aren't triggering.

Perhaps we can try to get them reworking separately, and decide then which of the two we want to keep?